app/docs/md/learn/concepts/styling/css.md

```diff
@@ -0,0 +1,93 @@
+---
+ title: Styling with CSS
+---
+
+Using CSS with Enhance is as easy adding CSS to a`<style>` tag.
+Just edit the `head.mjs` file in the root of your project and add a `<style>` tag. It's that easy.
+
+## Adding a style tag to your `head.mjs`
+
+Look for the `head.mjs` file in the root of your app.
+
+```
+app
+└── head.mjs
+```
+
+Add a style tag and start writing CSS.
+
+<doc-code filename='app/head.mjs'>
+
+```javascript
+export default function Head () {
+  return `
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title></title>
+      <link rel="icon" href="/_public/favicon.svg">
+      <style>
+        body {
+          background-color: rebeccapurple;
+        }
+      </style>
+    </head>
+`
+}
+```
+
+</doc-code>
+
+The benefits of adding styles to a style tag in the head is that the styles will be immediately available when the page loads which will help avoid FOUCing your users.
```

Comment:
Maybe say the below to remove one repetition of style(s) in the sentence?

"The benefits of adding a style tag in the head is that the styles will be immediately available…